models/openocean/optimism/openocean_optimism_trades.sql

```diff
@@ -1,5 +1,5 @@
 {{ config(
-    
+    tags = ['dunesql'],
```

Comment:
By the way, we can remove `dunesql` tags now, as all existing sqls are dune sql.

Reply:
Hello @Hosuke ,

I've implemented the changes you suggested and removed the dunesql tag. Please let me know if there are any additional changes required to the spell.